client/src/components/AddNewProfile.js

```diff
@@ -13,6 +13,7 @@ const AddNewProfile = () => {
     phone: "",
     gender: "",
     groups: [],
+    languages: "",
```

Comment:
Should there not be two separate fields here? For "language" and "other languages" (see comments below too) ?